Core/GDCore/Extensions/Metadata/DependencyMetadata.h

```diff
@@ -0,0 +1,89 @@
+/*
+ * GDevelop Core
+ * Copyright 2008-2016 Florian Rival (Florian.Rival@gmail.com). All rights
+ * reserved. This project is released under the MIT License.
+ */
+#ifndef DEPENDENCYMETADATA_H
+#define DEPENDENCYMETADATA_H
+#include <map>
+#include "GDCore/String.h"
+#include "GDCore/Project/PropertyDescriptor.h"
+#include "GDCore/Tools/Log.h"
+
+namespace gd {
+    /**
+     * \brief Contains a information about a Dependency (library) of an extension.
+     * \note Is subject to changes.
+     */
+    class GD_CORE_API DependencyMetadata {
+        public:
+            /**
+             * \brief Sets the name shown to users.
+             */
+            DependencyMetadata& SetName(const gd::String& name_) {
+                name = name_;
+                return *this;
+            };
+
+            /**
+             * \brief Sets the name used by the dependency manager to find the dependency.
+             * \example 
+             * \code
+             *  // For depending upon the NPM package is-thirteen
+             *  gd::DependencyMetadata dependencyMetadata = gd::DependencyMetadata();
+             *  dependencyMetadata.setExporterName("is-thirteen");
+             * \endcode
+             */
+            DependencyMetadata& SetExportName(const gd::String& exportName_) {
+                exportName = exportName_;
+                return *this;
+            };
+
+            /**
+             * \brief Set the version of the dependency to install. Use -1 for latest.
```

Comment:
"-1", or what about just not specifying anything (empty string)? Should be easy to change?